### ISSUE TITLE
App elements, in Arabic or Persian (and I think in other RTL languages too) stick to the side.

### ISSUE THREAD

**Issue by GithubUser232** [martykan/forecastie#664]

**Comment by robinpaulson** [comment]
Could you provide a screenshot? Thanks

**Comment by GithubUser232** [comment]
> Could you provide a screenshot? Thanks

It has been edited.

**Comment by robinpaulson** [comment]
What do you think about providing a mockup of how you think it should look? Maybe just a few of the elements, rather than all.

Could this mostly be solved by some padding at the right-hand edge of each element?

**Comment by igor-cali** [comment]
> What do you think about providing a mockup of how you think it should look? 

Yes, @GithubUser232 : could you highlight what are the misplaced/disliked elements in the two Arabic screenshots in the issue description?

Thanks.

**Comment by GithubUser232** [comment]
> What do you think about providing a mockup of how you think it should look? Maybe just a few of the elements, rather than all.
> 
> Could this mostly be solved by some padding at the right-hand edge of each element?

I really want to do that but I do not access to a computer which can do this things.

Yes, some padding at the right-hand edge of each element. That should be great.

**Comment by GithubUser232** [comment]
> > What do you think about providing a mockup of how you think it should look?
> 
> Yes, @GithubUser232 : could you highlight what are the misplaced/disliked elements in the two Arabic screenshots in the issue description?
> 
> Thanks.

As @robinpaulson said, It needs some padding at the right-hand edge of each element ,just like English version.
